### PR TITLE
fix for not loaded revision info

### PR DESCRIPTION
**Pull request by idradm**

### CODE REVIEW

**Comment by SebastianMarzjan** [comment]
Title of the comment will never be null in this context, so :+1:

**Comment by SebastianMarzjan** [comment]
Still looks good :)

File: extensions/wikia/PartnerFeed/PartnerFeed.body.php [hunk 1]
```diff
@@ -129,18 +129,23 @@ private function FeedRecentBlogComments ( $format ){
...
+							'url' => $oTitle->getFullURL(),
+							'date' => $commentPage[ 'level1' ]->mFirstRevision->getTimestamp(),
+							'author' => $tmpArticleComment[ 'author' ]->getName(),
+							'otherTags' => array(
+								'image' => AvatarService::getAvatarUrl( $commentPage[ 'level1' ]->mUser->getName(), $userAvatarSize )
+							)
+						);
+					} else {
+						\Wikia\Logger\WikiaLogger::instance()->error('Comment page loading failed', $commentPage);
+					}
```
Comment (macbre) [inline extensions/wikia/PartnerFeed/PartnerFeed.body.php]: Thanks for the fix!

Minor comment: store ``$commentPage[ 'level1' ]`` in a variable and use it in this block.

File: extensions/wikia/PartnerFeed/PartnerFeed.body.php [hunk 2]
```diff
@@ -129,18 +129,23 @@ private function FeedRecentBlogComments ( $format ){
...
-						'author' => $tmpArticleComment['author']->getName(),
-						'otherTags' => array(
-							'image' => AvatarService::getAvatarUrl( $commentPage['level1']->mUser->getName(), $userAvatarSize )
-						)
-					);
+					//make sure all data is loaded
+					if ( $commentPage[ 'level1' ]->load() ) {
+						$tmpArticleComment = $commentPage[ 'level1' ]->getData();
+
+						$feedArray[ ] = array(
```
Comment (hakubo) [inline extensions/wikia/PartnerFeed/PartnerFeed.body.php]: array() to []

Comment (hakubo) [inline extensions/wikia/PartnerFeed/PartnerFeed.body.php]: and $feedArray[ ] do we need space inside there?

**Pull request merged**